podcasts/Sharing/Clip/VideoExporter.swift

```diff
@@ -152,6 +158,10 @@ enum VideoExporter {
 
     // Part of Step 2 of video export to export the final file
     private static func exportFinalComposition(_ composition: AVComposition, to outputURL: URL, duration: Double, fileType: AVFileType, progress: Progress) async throws {
+        guard Task.isCancelled == false else {
```

Comment:
Nit: same here, can this one be just `if Task.isCancelled`?

Reply:
I prefer the guards because it forces the early return and seems easier to skim. What do you think? I could change to `guard !Task.isCancelled`.